src/community/clustering.jl

```diff
@@ -156,3 +156,55 @@ function global_clustering_coefficient(g::AbstractGraph)
     ntriangles == 0 && return 1.
     return c / ntriangles
 end
+
+"""
+    triangle_count(g::AbstractGraph)
+    Returns the total number of triangles in a graph
```

Comment:
reformat this to our standard:
```
"""
    function_name(args)

Return the ...
"""
```

(note: we use action verbs in the description.)